frontend/src/views/SecretOverviewPage/components/SecretOverviewTableRow/SecretOverviewTableRow.tsx

```diff
@@ -5,14 +5,16 @@ import {
   faEye,
   faEyeSlash,
   faKey,
-  faXmark
+  faXmark,
+  faCodeBranch
 } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { twMerge } from "tailwind-merge";
 
 import { Button, TableContainer, Td, Tooltip, Tr } from "@app/components/v2";
 import { useToggle } from "@app/hooks";
 import { DecryptedSecret } from "@app/hooks/api/secrets/types";
+import { SecretActionType } from "@app/views/DashboardPage/DashboardPage.utils";
```

Comment:
Kindly define it again here again rather than importing from another page as each page is independent

Reply:
Implemented